opentelemetry-sdk/CHANGELOG.md

```diff
@@ -11,6 +11,11 @@
   first interval instead of doing it right away.
   [#1970](https://github.com/open-telemetry/opentelemetry-rust/pull/1970)
   [#1973](https://github.com/open-telemetry/opentelemetry-rust/pull/1973)
+  - **Breaking** [#1985](https://github.com/open-telemetry/opentelemetry-rust/pull/1985)
+  Hide LogRecord attributes Implementation Details from exporters.
```

Comment:
```suggestion
  Hide LogRecord attributes Implementation Details from processors and exporters.
```

Same for processors also, right?

Reply:
yes, custom processors not part of otel-sdk crate.